app/views/drink_sections/index.html.erb

```diff
@@ -0,0 +1,98 @@
+<% require 'time' %>
+
+<div class = "menubody">
+
+    <h1>Drinks Menu for <%=Time.now.strftime("%A, %b %d")%></h1>
+    <div class = "center">
+
+      <h3 class = "toggle">~ CLICK TO HIDE MENU ITEMS ~</h3>
+      <h3 class = "toggle", style ="display: none">~ CLICK TO SHOW MENU ITEMS ~</h3>
+
+      <ul>
+        <% @drinks.each do |drink| %>
+        <li>
+          <a><h2><%= link_to drink.name, drink_path(drink.id) %></h2></a>
+        </li>
+          <%=drink.description %>
+
+
+          <div class = "hide">
+           <ul>
+            <% @drink_items.each do |drink_item| %>
+              <% if drink_item.drink_section_id == drink.id %>
+                <li>
+                  <span class="item"><%=drink_item.name%></span>, <span class="description"><%=drink_item.description%></span>,
+                  <span class="region"><%=drink_item.region%></span> <span class="price">(<%=drink_item.pricing%>)</span>
+                </li>
+              <% end %>
+            <% end %>
+           </ul>
+          </div>
+
+          <div class = "buttons">
+            <li>
+              <form action="/drinks/edit/<%=drink.id %>" method="POST">
+                <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
+                <input type="hidden" name="_method" value="GET" />
+                <input type="submit" value="Edit Drink Section" />
```

Comment:
Generally a plain ol link is best for a `GET` request, you can use css to style the link like a button if you want it to look the same as the delete button.